README.md

```diff
@@ -67,6 +67,28 @@ You can also update, re-arrange (for one-to-many relationships), and delete rela
 
 ![Rearrange Models](https://raw.githubusercontent.com/kirschbaum-development/nova-inline-relationship/master/screenshots/UpdateView.png "Rearrange Models")
 
+ To rearrange a model you should specify an `integer` field which will be used to sort models. For example, in the following code we will use a field named `weight` to store the sorting order for `Images`.
+ 
+ ```php
+ namespace App\Nova;
+ 
+ use Laravel\Nova\Fields\Image;
+ 
+ class BlogPost extends Resource
+ {
+     //...
```

Comment:
@navneetrai I don't think we need this extra class info here. The fields method is good enough